vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -306,6 +307,13 @@ func verifyRemoveSubnetValidatorTx(
 		return nil, false, err
 	}
 
+	if backend.Config.UpgradeConfig.IsEtnaActivated(currentTimestamp) {
+		_, _, err := chainState.GetSubnetManager(tx.Subnet)
+		if err != database.ErrNotFound {
+			return nil, false, ErrRemoveValidatorManagedSubnet
+		}
```

Comment:
Would it make sense to use the same check pattern in both places?
```suggestion
		_, _, err := chainState.GetSubnetManager(tx.Subnet)
		if err == nil {
			return nil, fmt.Errorf("%w: %q", ErrRemoveValidatorManagedSubnet, tx.Subnet)
		}
		if err != database.ErrNotFound {
			return nil, err
		}
```